src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -1,19 +1,80 @@
+import { Id } from '@colony/colony-js';
+import { defineMessages } from 'react-intl';
+
 import { type Domain } from '~types/graphql.ts';
+import { formatText } from '~utils/intl.ts';
 import { getTeamHexColor } from '~utils/teams.ts';
-import { type ChartData } from '~v5/shared/DonutChart/types.ts';
+
+import { type ReputationChartDataItem } from './types.ts';
 
 const WIDGET_TEAM_LIMIT = 5;
```

Comment:
This should be set to `4` as we want to display only the top 4 teams and for the remaining teams with reputation to show the `All other` slice/legend